openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
I've actually opted for an alternative since my last post. How about a new `NuTransportXS` class? I really hate adding in new properties/attributes since it seems like a sub-optimal hacky solution. Plus, I can foresee an explosion of them in the future with more complex transport corrections and the like.